app_changelog.go

```diff
@@ -0,0 +1,12 @@
+package versionbundle
+
+type AppsLog map[string][]Log
```

Comment:
```suggestion
type AppChangelogs map[string][]Changelog
```

Reply:
@rossf7 If we decide to go with `AppChangelog`, I think it should be same as below
`type AppChangelogs map[string][]AppChangelog`

right?